src/svgdom/util/style_stack_cache.cpp

```diff
@@ -7,101 +7,101 @@
 using namespace svgdom;
 
 namespace{
-class CacheCreator : virtual public svgdom::const_visitor{
+class cache_creator : virtual public svgdom::const_visitor{
 public:
-	std::map<std::string, finder::element_info> cache;
+	std::unordered_map<std::string, style_stack> cache;
 	
-	style_stack styleStack;
-	
-	void addToCache(const svgdom::element& e){
+	style_stack style_stack;
```

Comment:
add namespace to type name, i.e. `svgdom::style_stack style_stack` as the name of the variable and class name are same in this case, compiler probably will give an error.
Or, better solution is to avoid this kind of name clashing and use different name for the variable, e.g. `style_stack current_style_stack`, in this case the variable name is more descriptive and more precisely describes the meaning of the variable.